tests/foreman/ui/test_errata.py

```diff
@@ -728,17 +728,14 @@ def test_positive_show_count_on_chost_page(self):
                 result = self.contenthost.fetch_errata_counts(client.hostname)
                 for errata in ('security', 'bug_fix', 'enhancement'):
                     self.assertEqual(result[errata]['value'], 0)
-                    self.assertEqual(result[errata]['color'], 'black')
```

Comment:
Take a look at expected results, these colors should be verified according to docstring, why are you removing the checks?